antarest/study/storage/rawstudy/model/filesystem/ini_file_node.py

```diff
@@ -105,9 +105,9 @@ def _get(
         url = url or []
         kwargs = self._get_filtering_kwargs(url)
 
-        if self.config.zip_path:
-            with zipfile.ZipFile(self.config.zip_path, mode="r") as zipped_folder:
-                inside_zip_path = self.config.path.relative_to(self.config.zip_path.with_suffix("")).as_posix()
+        if self.config.archive_path:
+            with zipfile.ZipFile(self.config.archive_path, mode="r") as zipped_folder:
```

Comment:
here you should also handle 7z as i tested on my local env and it failed.
You should also write a test to reproduce:
1- Create a study with one area
2- Archive it
3- Use the endpoint GET /raw towards any Ini file.
4- It should succeed and currently it fails with an HTTP 500